Makefile

```diff
@@ -34,7 +34,7 @@ test:
 	go test ./...
 
 alltest: test
-	go test --integration ./...
+	go test ./... -integration
```

Comment:
Is here difference between `-integration` and `--integration` ?

Reply:
I've checked it and apparently there is not. I was confused since I've seen some of the Go commands using a single dash, and that's why I've been issuing these commands like that. But seeing that it works with a double dash, I'm sticking to that!

Reply:
yes apparently there is not. When  I was trying it I had locally other db discrepancies - so it confuses me.
thanks for changing it 👍